Doxyfile.in

```diff
@@ -618,7 +618,7 @@ CITE_BIB_FILES         =
 # The QUIET tag can be used to turn on/off the messages that are generated
 # by doxygen. Possible values are YES and NO. If left blank NO is used.
 
-QUIET                  = NO
+QUIET                  = YES
```

Comment:
But we lose the ability to diagnose which files it is or isn't checking for documentation (e.g. the recent RDM enums stuff I was looking at).